Framework/include/QualityControl/AggregatorRunnerConfig.h

```diff
@@ -0,0 +1,40 @@
+// Copyright 2019-2020 CERN and copyright holders of ALICE O2.
+// See https://alice-o2.web.cern.ch/copyright for details of the copyright holders.
+// All rights not expressly granted are reserved.
+//
+// This software is distributed under the terms of the GNU General Public
+// License v3 (GPL Version 3), copied verbatim in the file "COPYING".
+//
+// In applying this license CERN does not waive the privileges and immunities
+// granted to it by virtue of its status as an Intergovernmental Organization
+// or submit itself to any jurisdiction.
+
+///
+/// \file   AggregatorRunnerConfig.h
+/// \author Piotr Konopka
+///
+
+#ifndef QUALITYCONTROL_AGGREGATORRUNNERCONFIG_H
+#define QUALITYCONTROL_AGGREGATORRUNNERCONFIG_H
+
+#include <unordered_map>
+#include <string>
+
+namespace o2::quality_control::checker
+{
+
+struct AggregatorRunnerConfig {
+  std::unordered_map<std::string, std::string> database;
+  std::string consulUrl{};
+  std::string monitoringUrl{};
+  bool infologgerFilterDiscardDebug = false;
+  int infologgerDiscardLevel = 21;
```

Comment:
We could consider having a parent struct containing the common part of all the *Config struct. It is fine like that, in particular for readability though.

Reply:
Yes, this would be a nice improvement. That being said, I think we should prioritize moving to push config for now, but we could do this in the next iteration.